src/components/checkbox/checkbox.js

```diff
@@ -96,15 +96,31 @@ class Checkbox extends mixin(createComponent, initComponentBySearch, handles) {
     }
   }
 
+  setDisabled(value) {
+    if (value === undefined) {
+      throw new TypeError('setDisabled expects a boolean value of true or false');
+    }
+    if (value === true) {
+      this.element.setAttribute('disabled', true);
+    } else if (value === false) {
+      this.element.removeAttribute('disabled', true);
+    }
+    const container = this.element.closest('[data-contained-checkbox-disabled]');
+    if (container) {
+      this.element.parentElement.setAttribute('data-contained-checkbox-disabled', value);
```

Comment:
Did we want to change the attribute of the `container` here...?

Reply:
We need this for the styling for the *(input > label)* variant.

Reply:
OK thank you for clarifying!